corehq/apps/app_manager/views/modules.py

```diff
@@ -1352,18 +1353,20 @@ def _init_biometrics_identify_module(app, lang, enroll_form_id):
 
     form_name = _("Followup with Person")
 
+    output_tag = mark_safe(  # nosec: no user-input
+        "<output value=\"instance('casedb')/casedb/case[@case_id = "
+        "instance('commcaresession')/session/data/case_id]/case_name\" "
+        "vellum:value=\"#case/case_name\" />"
+    )
+
     context = {
         'xmlns_uuid': generate_xmlns(),
         'form_name': form_name,
         'lang': lang,
-        'placeholder_label': mark_safe(_(
+        'placeholder_label': format_html(_(
```

Comment:
I can see the value of reducing the footprint of `mark_safe`, but why is the previous code unsafe, given that it doesn't include user input?

Reply:
Correct, it's not unsafe. It was done for two reasons:
- Going forward, the practice should be, if you're building up an HTML string, you use `format_html`
- Bandit will complain about any usages of mark_safe, regardless if its safe or not. We can ignore those lines with a `#nosec` comment, like above, but I think those should be used with as limited a scope as possible.

Reply:
Cool, sounds good.